xbmc/epg/EpgContainer.cpp

```diff
@@ -334,13 +333,13 @@ void CEpgContainer::Process(void)
   }
 }
 
-CEpg *CEpgContainer::GetById(int iEpgId) const
+std::shared_ptr<CEpg> CEpgContainer::GetById(int iEpgId) const
```

Comment:
We use typedefs for situations like these elsewhere in the code, i.e. `CEpgPtr`. Can you add that?